x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -576,10 +592,17 @@ export function getTestUrl(
   job: Job | DataFrameAnalyticsConfig,
   customUrl: MlUrlConfig,
   timeFieldName: string | null,
-  currentTimeFilter?: EsQueryTimeRange
+  currentTimeFilter?: EsQueryTimeRange,
+  isPartialDFAJob?: boolean
 ) {
-  if (isDataFrameAnalyticsConfigs(job)) {
-    return getDataFrameAnalyticsTestUrl(job, customUrl, timeFieldName, currentTimeFilter);
+  if (isDataFrameAnalyticsConfigs(job) || isPartialDFAJob) {
+    return getDataFrameAnalyticsTestUrl(
+      job as DataFrameAnalyticsConfig,
+      customUrl,
+      timeFieldName,
+      currentTimeFilter,
```

Comment:
The time range used when hitting 'test' after adding a URL always seems to go `Last 15 minutes` (maybe that's the default time filter?)- can it use the same logic that seems to work when creating the URL in the popover when calculates the time range based on the interval and a doc from the source index? 

Also tested this in the anomaly detection wizard, and seems to be broken there even in the popover - neither the time range or query entities are getting passed correctly.

Reply:
Yes, that's the default time when a custom time range or an interval time range has not been provided.
To clarify - when you set an interval or a custom time range - is it still always going to `Last 15 minutes` ?

Reply:
In the DFA wizard after adding the custom URL if I hit 'test' I am finding:

- Interval time range: shows Last 15 minutes
- Fixed time range: Works

Before adding, when I test from the popover, the interval time range works as expected.

Reply:
Okay, fixed in d943a422d795e15cd179c9c2686be54200c26bf9
cc @peteharverson

Reply:
Tested latest and the time range is getting set correctly for me.